src/mixs/schema/mixs.yaml

```diff
@@ -16354,8 +16358,13 @@ classes:
         string_serialization: '{float} {unit};{period};{interval};{period}'
         recommended: true
     class_uri: MIXS:0016018
+    aliases:
+      - MIxS-Ag (agriculture)
+    annotations:
+      use_cases: Agricultural Microbiomes Research Coordination Network, model cropping and plant systems focused on agricultural plant and soil microbe; microbiome studies in agricultural sites; long-term ecological research in croplands; eDNA in manure samples; describing agricultural microbiome studies
   Air:
-    description: air extension
+    description: >
+      A collection of terms appropriate when collecting and sequencing obtained from a gaseous environment.
```

Comment:
suggested change: A collection of terms appropriate when collecting and sequencing **samples** obtained from a gaseous environment.